projects/basisstatistiek.md

```diff
@@ -0,0 +1,49 @@
+---
+abstract: Descriptive statistics about the population and real estate of Amsterdam
+reusability: 1
+---
+
+# Basisstatistiek
+
+**This project in only available in Dutch**
+
+All documentation, scripts and syntaxes, and datamodels (refered to as `code`) used for generating
+descriptive statistics about the population and real estate of Amsterdam.
+
+User friendly documentation can be found on our [documentation](https://basisstatistiek.amsterdam.nl) website.
```

Comment:
This website is only available within the internal firewall of the City of Amsterdam, perhaps this can either be opened or that needs to be noted.

Reply:
Nog wel onder voorbehoud van een Amsterdamse huisstijl. Ik vond het teveel gedoe om dat voorbehoud er nu in te zetten om het later weer te moeten aanpassen zodra hij wel openbaar is.